wagtail/wagtailsnippets/tests.py

```diff
@@ -131,6 +131,17 @@ def test_simple(self):
         response = self.get()
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'wagtailsnippets/snippets/create.html')
+        self.assertNotContains(response, '<a href="#advert" class="active">Advert</a>')
```

Comment:
You should pass `html=True` to `assertNotContains`, otherwise it does a plain-text comparison:

```python
self.assertNotContains(response, '<a href="#advert" class="active">Advert</a>', html=True)
```

This also applies to the `assertContains` calls below.